scripts/approvetokens.ts

```diff
@@ -0,0 +1,9 @@
+import { ethers } from "hardhat";
+async function main() {
+    let token = await ethers.getContractAt("ERC20MockToken", ""/**Contract Address */);
```

Comment:
need to pass the address parameter. Better to be done in [task](https://hardhat.org/hardhat-runner/docs/advanced/create-task) format